src/platforms/common/user-feedback/index.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: Set Up User Feedback
+sidebar_order: 6000
+redirect_from:
+  - /learn/user-feedback/
+description: "Learn more about collecting user feedback when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues."
+---
+
+<PlatformSection supported={["javascript"]} notSupported={["javascript.electron", "node", "react-native"]}>
+<Include name="feature-stage-beta-user-feedback.mdx" />
+
+Sentry now allows you to collect user feedback from anywhere inside of your application, without requiring an error event to occur. This requires a minimum SDK version of [#TODO](#TODO). The previous [Crash-Report modal](#crash-report-dialog) still exists to handle user feedback that is associated with an error event.
+
+## User Feedback Widget
+
+Our embeddable JavaScript widget allows your users to submit feedback from anywhere inside of your application. This differs from the [Crash-Report modal](#crash-report-dialog) which is reactive feedback that is specifically tied to an error event.
+
+!! #Insert image of widget here# !!
+
+### Pre-requisites
+
+<PlatformContent includePath="user-feedback/pre-requisites" />
+
+### Installation
+
+<PlatformContent includePath="user-feedback/install" />
+
+### Set Up
+
+To set up the integration, add the following to your Sentry initialization. There are many options you can pass to the integration constructor. See the [configuration documentation](/platforms/javascript/user-feedback/configuration/) for more details.
+
+<SignInNote />
+
+<PlatformContent includePath="user-feedback/setup" />
+
+By default, this will insert the widget into the bottom right corner of your website. You're free to customize nearly every aspect of the widget, including replacing it completely with your own UI.
+
+</PlatformSection>
+
+<PlatformSection notSupported={["javascript"]}>
+  When a user experiences an error, Sentry provides the ability to collect
+  additional feedback. You can collect feedback according to the method
+  supported by the SDK.
+</PlatformSection>
+
+<PlatformSection supported={["ruby"]} notSupported={["ruby.rails"]}>
+
+While this feature isn't currently supported for Ruby or most of its frameworks, it is supported for [Rails](/platforms/ruby/guides/rails/user-feedback/).
+
+</PlatformSection>
+
+<PlatformSection supported={["native"]}>
+
+**The user feedback feature is not currently supported for this SDK.**
+
+</PlatformSection>
+
+<PlatformSection supported={["javascript", "java", "apple", "android", "dart", "flutter", "unreal", "react-native", "kotlin-multiplatform"]}>
+
+## User Feedback API
+
+The user feedback API provides the ability to collect user information when an event occurs. You can use the same programming language you have in your app to send user feedback. In this case, the SDK creates the HTTP request so you don't have to deal with posting data via HTTP.
```

Comment:
Maybe it's obvious to others, but I'd like more clarity here. How does the user feedback API connect to the widget? Can you use the widget without the API or do you need to set up both? The way it's described seems kinda similar to the crash report modal

Reply:
@vivianyentran The crash-report modal Sentry provides is only for Web platforms. You can still leverage the User Feedback functionality by calling the API for these other non-web platforms.

Reply:
@vivianyentran Thoughts on the new text?

> The user feedback API allows you to collect user feedback while utilizing your own UI. You can use the same programming language you have in your app to send user feedback. In this case, the SDK creates the HTTP request so you don't have to deal with posting data via HTTP.